battleship-client/src/main/java/com/epam/solid/nie/client/communication/SocketClient.java

```diff
@@ -28,7 +37,8 @@ public boolean run() throws IOException {
 
         String fromServer;
         while (!(fromServer = in.readLine()).equals("Provide ships")) {
```

Comment:
Referring to "Provide ships",  if a String is used as a message, I think it should be replaced by an enum